bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/XMLNonElementItem.java

```diff
@@ -137,7 +138,6 @@ public XMLValue getItem(int index) {
 
     @Override
     public Object copy(Map<Object, Object> refs) {
-        // XMLContentHolderItem is immutable
         return this;
     }
```

Comment:
This logic only applies to XMLText.java now right? Shall we remove this implementation and implement `copy()` XMLText.java instead?

Reply:
+1